fabulaws/library/wsgiautoscale/api.py

```diff
@@ -824,6 +829,15 @@ def upload_supervisor_conf(run_update=True):
         use_jinja=True,
         template_dir=env.templates_dir,
     )
+    _upload_template(
```

Comment:
It doesn't look like there's actually templating going on (there are no template tags in the template gunicorn-entrypoint.sh). Any reason not to just use fabric's `put`?

Reply:
Good question. I am mostly just being lazy / not wanting to maintain custom versions of both `put` and `upload_template`. Your commend did make me realize the `context` and `use_jinja2` are not needed though, so I've cleaned that up and moved `chmod` to within `_upload_template()`.

Thanks for the review!